modules/apps/batch-planner/batch-planner-test/src/testIntegration/java/com/liferay/batch/planner/batch/engine/broker/test/BatchEngineBrokerTest.java

```diff
@@ -1158,16 +1158,35 @@ private List<String> _normalizeValues(List<String> values) {
 		List<String> normalizedValues = new ArrayList<>(values.size());
 
 		for (String value : values) {
-			if (Validator.isNotNull(value) &&
+			if (Validator.isNotNull(value) && !value.isBlank() &&
 				_htmlTagPattern.matcher(
 					value
 				).find()) {
 
-				value = _htmlBreakPattern.matcher(
-					value
-				).replaceAll(
-					"$1\n\n$3"
-				);
+				if (_HTML_NORMALIZATION_MODE ==
```

Comment:
Is possible to use the Sanitizer class of Liferay instead of performing all of these changes by ourselves? I don't know if it makes sense to know all he rules we need to apply to transform a CSV